src/components/organisms/modals/TooltipModal/TooltipModal.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  AppearanceTypes,
+  IconPositioningTypes,
+  SizeTypes,
+} from 'components/molecules/Button/Button'
+import ModalBase, {
+  ButtonPositionTypes,
+  TitleFontTypes,
+} from 'components/organisms/ModalBase/ModalBase'
+import classes from './styles.module.scss'
+
+export interface TooltipModalProps {
+  title?: string
+  textButtonContent?: string
+  modalContent?: string
+  href?: string
+  isModalOpen?: boolean
+  closeModal: () => void
+}
+
+const TooltipModal: FC<TooltipModalProps> = ({
+  title,
+  textButtonContent,
```

Comment:
**[non-blocker]**

Right now there is no point to do it, but once we add the page for the FAQ or (Kasutajajuhend) page, we should also add default values for textButtonContent and href, so it would navigate to the FAQ page by default